src/Microsoft.SqlTools.Hosting/Utility/CommandOptions.cs

```diff
@@ -94,20 +95,25 @@ public CommandOptions(string[] args)
         public string Locale { get; private set; }
 
         /// <summary>
+        /// Name of service that is receiving command options
+        /// </summary>
+        public string ServiceName { get; private set; }
+
+        /// <summary>
         /// Get the usage string describing command-line arguments for the program
         /// </summary>
         public string Usage
         {
             get
             {
                 var str = string.Format("{0}" + Environment.NewLine +
-                    "Microsoft.SqlTools.ServiceLayer.exe " + Environment.NewLine +
+                    ServiceName + " " + Environment.NewLine +
                     "   Options:" + Environment.NewLine +
                     "        [--enable-logging]" + Environment.NewLine +
                     "        [--log-dir **] (default: current directory)" + Environment.NewLine +
                     "        [--help]" + Environment.NewLine +
                     "        [--locale **] (default: 'en')" + Environment.NewLine,
-                    ErrorMessage);
+                    (object) ErrorMessage);
```

Comment:
Is `(object)` necessary? It wasn't there before

Reply:
whoops, not sure how that got introduced. Will remove